web/src/components/organisms/DatePicker.jsx

```diff
@@ -0,0 +1,12 @@
+import { Datepicker } from "flowbite-react";
+
+
+const DatePicker = () => {
+    return (
+        <div>
+            <Datepicker autoHide={false} />;
+        </div>
+    )
```

Comment:
I think better to use react fragment 
```suggestion
    return (
        <>
            <Datepicker autoHide={false} />;
        </>
    )
```